src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Start using the new synthetic monitoring runtime
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+Synthetic monitoring has a new runtime that affects your public monitors. By converting from the legacy runtime to the new runtime, you improve the backend environment that your public monitors execute from. New synthetic monitors default to using the new runtime. If you've created a synthetic monitor using the legacy runtime, there are extra [configuration steps](#runtime-convert).
+
+Making the switch gives you these features:
+
+* Access to newer versions of Google Chrome, so your scripted browser monitors use browser versions similar to the ones used by your customers. 
+* Access to the updated scripted API runtime that uses `Got` node.js library  instead of the deprecated `Request` library. 
+* Support for async / await syntax. This simplifies script creation and maintenance, diverging from the old runtime’s `.then` promise chaining.
+* The ability for your monitors to treat objects like frames and find objects inside the shadow DOM.
```

Comment:
I think we should simplify this and say "Better support for shadow DOM objects". While interacting with these objects feels somewhat like working with frames, this context may confuse our users.